client/my-sites/theme/live-preview-button/index.tsx

```diff
@@ -26,8 +27,11 @@ export const LivePreviewButton: FC< Props > = ( { themeId, siteId } ) => {
 	}
 
 	return (
-		<Button onClick={ () => dispatch( livePreview( themeId, siteId, 'detail' ) ) }>
-			{ translate( 'Preview & Customize' ) }
-		</Button>
+		<>
+			<TrackComponentView eventName="calypso_live_preview_button_impression" />
```

Comment:
The corresponding click event is called `calypso_block_theme_live_preview_click`. Should we rename the event to something like `calypso_block_theme_live_preview_impression`?

Reply:
I kind of don't like that name but changed it anyways 🤷‍♂️